pkg/catalog/catalog.go

```diff
@@ -1794,6 +1794,7 @@ func (c *Catalog) WriteRange(ctx context.Context, repositoryID string, params Wr
 	stagingToken := params.StagingToken
 	skipped := it.GetSkippedEntries()
 	if len(skipped) > 0 {
+		c.log.Warning("Skipped count:", len(skipped))
```

Comment:
Remove 2 lines down and add
```suggestion
		c.log.WithField("skipped_amount", len(skipped)).WithField("token", stagingToken).Warning("Writing skipped entries to staging token")
```